Dockerfile.konflux

```diff
@@ -0,0 +1,111 @@
+# Copyright 2021 IBM Corporation
```

Comment:
rename to Containerfile?

Reply:
We trying to keep standardized naming across all components as Dockerfile.konflux.